docs/en/stock-management/movement.shipments.md

```diff
@@ -0,0 +1,111 @@
+&raquo; [Home](../index.md) / [Stock Management](./index.md) / [Stock Movements](./movement.md) / Shipments
+
+# Shipments
+
+When stock is transferred from one depot to another, it is called a "shipment".  This could be involve collecting stock in one depot and shipping it via truck to another depot in an entirely different location.  Or it could simply be taking some stock from one depot to another in the same building.   For instance, purchasing and receiving stock from external suppliers is often done through a primary depot, while a pharmacy is also considered a "depot" in BHIMA. So once stock is received in the primary depot, transferring it to a pharmacy in the same building would still be considered a shipment.
+
+Shipments are restricted to transferring stock from one depot to another.   These involve stock exits (from the sending depot) and stock entry (into the receiving depot).   Technically, transferring stock from one depot to another could be done by directly using the Stock Exit and Stock Entry interfaces and not directly using the Shipping interface.  However, users should be aware that even if the shipment interface is not directly used, a shipment is still created when stock is exited from one depot and sent to another.  Other types of stock exits (such as exit to "Loss") or stock entries (such as entry by "Integration") do not involve BHIMA "Shipments".
+
+
+## The BHIMA Shipments interface
+
+In the BHIMA interface, there is  "Shipments" menu entry in the top-level menu:
+<img src="./images/shipments-menu.png" alt="Main shipments menu" align="right" width="200">
+
+There are two entries under the "Shipments" menu:
+1. New Shipment - Create a new shipment
+2. Shipment Registry - Display shipments with status and actions that can be performed on them.
+
+## The Lifecycle of a Shipment
+Shipments can be the following states:
+- **Empty** - the order has been created but contains no stock
+- **At depot** - the shipment has been created but not finalized
+- **Ready for shipment** - The shipment packing list is finalized
+- **In transit** - The shipment is in
+- **Partial** - Partially Received by the receiving depot
+- **Complete** - The order has been completed (sent and received)
+- **Delivered** - The order has been received but not unpacked or entered into stock
+- **Lost** - The order was lost (typically lost in transit)
+
+Shipments go through several phases during their lifetime (these phases are related to the states listed above):
+1. **Creation** - In this step, a shipment is created including information about
+   - the sending and receiving depots
+   - expected arrival date, and
+   - a packing list of stock to be shipped.
+During creation, it is still possible to update the packing list of items in the shipment. Once the shipment container is completely packed, the warehouse proceeds to the next step."
```

Comment:
Just one nitpick - you have a quotation mark at the end of this line, likely as a copy and paste error.  Could you fix it?

Reply:
Fixed